manifests/package.pp

```diff
@@ -0,0 +1,3 @@
+class mcollective::package {
+  ensure_packages($mcollective::package_name, { ensure => $mcollective::package_ensure })
```

Comment:
no way am I using these awful functions.

If you want the module to manage the package let it do so - ensure_packages exist to allow multiple modules to manage the same package and I am just not going to waste my time supporting that scenario :)

So lets make this `package{$mcollective::package: ...}`